src/components/dashboard/Reason.js

```diff
@@ -33,19 +82,167 @@ const SendReason = (props: AmountProps) => {
     })
   }, [restState, reason, screenState.nextRoutes, params])
 
+  const handleCategoryBoxOnPress = category => {
+    // set category and enable next button
+    setScreenState({ category, isDisabledNextButton: false })
+
+    // fire event
+    fireEvent(PAYMENT_CATEGORY_SELECTED, {
+      action: screenState.action,
+      amount: screenState.amount,
+      category: PaymentCategory.labelOf(category),
+      reason: screenState.reason,
```

Comment:
i dont think we should send reason, that's private

Reply:
done